pubspec.yaml

```diff
@@ -29,6 +29,7 @@ dependencies:
   path: ^1.6.2
   quiver: ^3.0.0
   ffi: ^2.0.1
+  image: ^4.0.16
```

Comment:
Where'd this dependency come from for the text classifier?

Reply:
No, it comes from the [Image Classification](https://github.com/am15h/tflite_flutter_helper/tree/master/example/image_classification) I worked in the [integration](https://github.com/mlopez0/flutter-tflite/tree/image_classification_example) with this new plug-in

Reply:
In my opinion it should be in the core part of the plugin, that's why [I create this PR](https://github.com/tensorflow/flutter-tflite/pull/26)

If you check the original example the package is used in image_clasification and [object_detection](https://github.com/am15h/object_detection_flutter/blob/master/pubspec.yaml)

Reply:
Hey @mlopez0 if you can just nix this pubspec.yaml change, I think everything else should be good to go. Thanks!

Reply:
fixed!